ocaml/idl/datamodel.ml

```diff
@@ -983,6 +983,10 @@ let _ =
     ~doc:"This operation cannot be performed because the system does not manage this VDI" ();
   error Api_errors.vdi_not_in_map [ "vdi" ]
     ~doc:"This VDI was not mapped to a destination SR in VM.migrate_send operation" () ;
+  error Api_errors.vdi_out_of_space [ "vdi" ]
+    ~doc:"This VDI is out space to write any more data" () ;
+  error Api_errors.vdi_not_formatted [ "vdi" ]
+    ~doc:"This VDI is not formatted to read or write any data" () ;
```

Comment:
Out of curiosity, when would we expect this to happen and what would we tell the user to do about it?